Felienne/hedy
# [BUG FIX] Fixes incorrect pop-up within tutorial

**Description**
This PR fixes a very small issue where one step of the tutorial was accidentally still hard-coded with a step number. Due to implementation an additional step the text of the step was repeated. This is fixed by making it dependent on the `current_step` variable as well.
 
**Fixes**
No related issue.

**How to test**
Verify that the "save & share" step in the tutorial is correctly displayed whereas currently the "quiz" step is shown twice.
